cypress/components/batch-selection/batch-selection.cy.tsx

```diff
@@ -12,13 +10,26 @@ import {
   batchSelectionButtonsByPosition,
 } from "../../locators/batch-selection/index";
 
+type BATCH_SELECTION_COLOR_SCHEME_TYPE =
+  | "dark"
+  | "light"
+  | "white"
+  | "transparent";
```

Comment:
```suggestion
```

To save you from having to redefine the types for particular component props, you can use an [indexed access type](https://www.typescriptlang.org/docs/handbook/2/indexed-access-types.html) to get the type of a specific prop from the component's prop interface:

`BatchSelection["colorTheme"]`

This should help reduce the verbosity of the tests